src/github/credentials.ts

```diff
@@ -89,32 +101,35 @@ export class CredentialStore {
 
 		// the remote url might be http[s]/git/ssh but we always go through https for the api
 		// so use a normalized http[s] url regardless of the original protocol
-		const normalizedUri = remote.gitProtocol.normalizeUri();
-		const host = `${normalizedUri.scheme}://${normalizedUri.authority}`;
+		const {scheme, authority} = remote.gitProtocol.normalizeUri();
```

Comment:
we should add the tslint rule for this, in other places we have whitespace around curlys when deconstructing

Reply:
Changed this instance. I'll add the lint rule and clean up anything it finds in a separate CL.